x-pack/metricbeat/module/googlecloud/stackdriver/compute/metadata.go

```diff
@@ -27,6 +27,7 @@ func NewMetadataService(projectID, zone string, region string, opt ...option.Cli
 		region:        region,
 		opt:           opt,
 		instanceCache: common.NewCache(30*time.Second, 13),
+		logger:        logp.NewLogger("metadata"),
```

Comment:
I'm thinking if the user facing log should not say something like the name of the metricset instead of the Go package

Reply:
Good point, thanks! Fixed.